dashboard/src/babelTransform.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable */
+"use strict";
+
+const babelJest = require("babel-jest");
+
+const hasJsxRuntime = (() => {
+  if (process.env.DISABLE_NEW_JSX_TRANSFORM === "true") {
+    return false;
+  }
+
+  try {
+    require.resolve("react/jsx-runtime");
+    return true;
+  } catch (e) {
+    return false;
+  }
+})();
+
+module.exports = babelJest.createTransformer({
+  presets: [
+    [
+      require.resolve("babel-preset-react-app"),
+      {
+        runtime: hasJsxRuntime ? "automatic" : "classic",
+        // Temporary workaround until this issue get solved:
```

Comment:
Is it the case that we're not even using Flow since we don't have raw js files that require static type checking, but rather are using typescript? If that's true, perhaps a comment saying as much here?

Reply:
Great, will do!
BTW, I've filed an additional issue in CRA: https://github.com/facebook/create-react-app/issues/11159
For us, it's fine either CRA adding a wat to disable Flow or Babel actually fixing the issue.

Reply:
Babel has just fixed the issue :tada:  https://github.com/babel/babel/issues/13520